documentation/modules/ROOT/pages/operations/kubernates.adoc

```diff
@@ -0,0 +1,370 @@
+[id="deploying-debezium-on-kubernates"]
+= Deploying {prodname} on Kubernates
```

Comment:
```suggestion
= Deploying {prodname} on Kubernetes
```

Same in the file name itself and elsewhere in this file. Best do a search/replace everywhere.